### ISSUE TITLE
doesn't work with python3

### ISSUE THREAD

**Issue by hasufell** [renpy/pygame_sdl2#6]
Old pygame (at least the 1.9.2 pre-releases) compiled fine with python3. pygame_sdl2 currently doesn't. The first error I get is invalid syntax for "print".

**Comment by renpytom** [comment]
This is correct, and something that needs to be addressed relatively soon.

**Comment by renpytom** [comment]
I've addressed this. The current HEAD branch should be able to build and run on python3. Please let me know if you have problems - those are now proper bugs, rather than something we didn't get to.

**Comment by Elinvention** [comment]
```
$ python3 setup.py install
pygame_sdl2.error is out of date.
pygame_sdl2.color is out of date.
pygame_sdl2.controller is out of date.
pygame_sdl2.rect is out of date.
warning: src/pygame_sdl2/rect.pyx:402:8: Unreachable code
pygame_sdl2.rwobject is out of date.
pygame_sdl2.surface is out of date.

Error compiling Cython file:
------------------------------------------------------------
...
                self.surface.pitch)

        self.unlock()

    def set_colorkey(self, color, flags=0):
        print color
             ^
------------------------------------------------------------

src/pygame_sdl2/surface.pyx:381:14: Syntax error in simple statement list

Command '['cython', '-3', '-Iinclude', '-Igen3', '-a', 'src/pygame_sdl2/surface.pyx', '-o', 'gen3/pygame_sdl2.surface.c']' returned non-zero exit status 1
```

Should I open a new issue?

**Comment by renpytom** [comment]
No need. I just fixed it with acc3178. 

If it happens again, opening a new issue to track it would be good, as I can close it as part of the command. Hopefully it won't happen again, though.
